### PR TITLE
Complete get mentor list

### PR DESCRIPTION
**Pull request by SarahhhC**
This feature completed!
Tests were not completed yet, but I'm gonna find how to test well controllers...
Give me comments!

@ethkim @nlpenguin @ifndefdeadmau5 @djflexible @lyw0149

### CODE REVIEW

File: src/server/controllers/users.controller.js [hunk 1]
```diff
@@ -21,22 +21,19 @@ export function getAll(req, res, next) {
...
-  User.find({ 'email' : { $ne : 'session@yoda.com' }},
-    // TODO: Longer term, we should migrate to a UserSummary object
-    // that contains subset of fields. For now, we return only the following
-    // fields from user object:
-    //
-    // - userId
-    // - name
-    // - field
-    { userId: 1, email: 0, name: 1, age: 0, field: 1, region: 0, skills: 0, regDate: 0 }, (err, doc) => {
+  if (typeof req.session.access_token != 'undefined' && req.session.access_token == req.query.access_token) {
```
Comment (ethkim) [inline src/server/controllers/users.controller.js]: break line after &&.
Also, I think the styleguide states we should use === and !== over == and != respectively. I haven't been paying attention to it till now, but we should start doing so. :)

See https://github.com/airbnb/javascript#comparison--eqeqeq

File: src/test/controllers/users.controller.test.js [hunk 2]
```diff
@@ -31,6 +31,7 @@ describe('Test for users.controller', function() {
   });
 
   describe('#auth()', function() {
+    // TODO: auth test modify
```
Comment (ethkim) [inline src/test/controllers/users.controller.test.js]: Not clear what this TODO means. Please elaborate more.

**Comment by ethkim** [comment]
Thanks. A few style comments, but other than that, LGTM.

**Comment by ethkim** [comment]
LGTM

File: src/server/controllers/users.controller.js [hunk 3]
```diff
@@ -21,22 +21,19 @@ export function getAll(req, res, next) {
...
-  User.find({ email: { $ne: 'session@yoda.com' } },
-    // TODO: Longer term, we should migrate to a UserSummary object
-    // that contains subset of fields. For now, we return only the following
-    // fields from user object:
-    //
-    // - userId
-    // - name
-    // - field
-    { userId: 1, email: 0, name: 1, age: 0, field: 1, region: 0, skills: 0, regDate: 0 }, (err, doc) => {
+  if (typeof req.session.access_token !== 'undefined' && req.session.access_token == req.query.access_token) {
```
Comment (ifndefdeadmau5) [inline src/server/controllers/users.controller.js]: Use '===' instead of '=='

**Comment by ifndefdeadmau5** [comment]
LGTM

**Comment by ethkim** [comment]
LGTM
Pls wait till travis says this is ok.

**Pull request merged**